tests/address_bar_and_search/conftest.py

```diff
@@ -3,25 +3,26 @@
 
 @pytest.fixture()
 def suite_id():
-    return ("18215", "Address Bar and Search 89+")
+    return "18215", "Address Bar and Search 89+"
```

Comment:
just curious, why did you decide to not have the values in a tuple? I normally see it like that in other conftests